subgraph/src/entities/approval-day-data.ts

```diff
@@ -4,7 +4,7 @@ import { ZERO } from '../constants';
 import { loadToken } from './token';
 
 export function loadApprovalDayData(day: i32, token: Address): ApprovalDayData {
-  const id = 'ADD-'
+  let id = 'ADD-'
```

Comment:
why const replaced with let?

Reply:
```
⠋ Compile subgraphWARNING AS206: Compiling constant with non-constant initializer as mutable.

   const id = address.toHexString();
```

the typescript web assembly complains on builds. this is pretty consistent with most subgraph implementations
https://github.com/enzymefinance/enzyme-subgraph/blob/main/src/mappings/GuaranteedRedemption.ts

example from thegraph protocol summit for best practices